spec/apollo-server-spec.yaml

```diff
@@ -0,0 +1,56 @@
+swagger: "2.0"
+info:
+  description: "This is the backend component API. Visit the link for an interactive playground to execute requests."
+  version: "1.0.0"
+  title: "Road Rules Search Engine"
+tags:
+  - name: "Backend service"
+schemes:
+  - "https"
```

Comment:
you will only get `http` out-of-the-box unless you do the work to set up SSL for your service, so I would change this to `http` for now